index.rst

```diff
@@ -6,24 +6,219 @@
+
+
+Hardpoint Breakaway Test
+========================
+
+The active support system of the M1M3 includes six axial hard point actutators in the hexapod configuration. :cite:`2018SPIE10700E..3GD`
+These hard point actuators should minimize forces during slews at any TMA position and be kept under the breakaway limit. 
+The breakaway limits for each hard points should happen in the range of -4420 to -3456 N for retraction and 2981 to 3959 N for extension.
+The followings steps are performed during the individual hard point breakaway test.
+
+1. Move hard point in negative (increasing tension) direction until either a low limit switch is actuated
+
+2. Move hard point in positive (increasing compression) direction till the high limit switch is triggered
+
+3. Move hard point downwards (increasing tension) until the low limit switch is hit
+
+4. Move hard point back to reference position
+
+5. Wait for the hard point to reach the reference position
+
+
+Note that for tests only full travel, from low limit to high limit and back, performed in steps 2 and 3 shall be considered. Other hard point movements do not guarantee a full hard point motion range will be examined.
+
+
+List of Hardpoint Breakwaway Test
+=================================
+
+.. _table-label:
+
+.. table:: Table 1. List of the Hardpoint Breakaway Test
+
+
+    +----------+--------+----------------------+----------+ 
+    | elevation| azimuth| Start Time           | SALIndex |
+    +----------+--------+----------------------+----------+ 
+    | (deg)    | (deg)  | (YYYY-MM-DDTHH:MM:SS)|          |
+    +==========+========+======================+==========+ 
+    | 0        | -29.69	| 2023-05-30T21:26:51  | 100056   |
+    +----------+--------+----------------------+----------+ 
+    | 1        | -29.69 | 2023-05-30T22:40:34  | 100057   |
+    +----------+--------+----------------------+----------+ 
+    | 5        | -29.69 | 2023-05-31T00:00:10  | 100058   |
+    +----------+--------+----------------------+----------+ 
+    | 10       | -29.69 | 2023-05-31T01:03:26  | 100059   |
+    +----------+--------+----------------------+----------+ 
+    | 20       | 153    | 2023-05-27T02:49:55  | 100036   |
+    +----------+--------+----------------------+----------+ 
+    | 20       | 153    | 2023-05-30T08:26:34  | 100047   |
+    +----------+--------+----------------------+----------+ 
+    | 40       | 153    | 2023-05-26T02:23:28  | 100034   |
+    +----------+--------+----------------------+----------+ 
+    | 89.95    | 153    | 2023-06-20T03:11:00  | 100038   |
+    +----------+--------+----------------------+----------+ 
+    | 90       | -29.69 | 2023-05-31T05:44:14  | 100061   |  
+    +----------+--------+----------------------+----------+ 
+
+
+
+Hard Point Breakaway Test at el=90
+==================================
+
+These are results from hard point breakaway tests when the TMA is positioned at el=90 deg, az=-29.69 deg.  
```

Comment:
Could you add a summary or interpretation of the test results here? Something like: they all look good at 90º because of X, Y, and Z. Similarly, if they fail, explain quickly why.

Reply:
I put "Measured forces on all of hard points look working properly because breakaway happened in the range of the requirement (Tension: -4420 - -3456N, Compression: 2981 - 3959N)." at the end of this sentence. But I think we'd better put more detailed explanations or descriptions in the section for the "A general analysis" later.